app/views/api/v1/conversations/partials/_conversation.json.jbuilder

```diff
@@ -22,7 +22,7 @@ if conversation.messages.first.blank?
 elsif conversation.unread_incoming_messages.count.zero?
   json.messages [conversation.messages.includes([{ attachments: [{ file_attachment: [:blob] }] }]).last.try(:push_event_data)]
 else
-  json.messages conversation.unread_messages.includes([:user, { attachments: [{ file_attachment: [:blob] }] }]).last(10).map(&:push_event_data)
```

Comment:
This was not useful, so had to remove it

Reply:
Can you check all the pages, may be add backend/frontend specs if there are any for JSON expectations?

Reply:
Checked those, and the response will be the same, the changes only affect under-the-hood ORM operations. I tested a bunch of cases, and the existing unit tests cover most of the scenarios